### PR TITLE
Varnish

### PR DESCRIPTION
**Pull request by tosan88**
Varnish &
+ PAM version increment (commit from pre-prod not merged to XP)

### CODE REVIEW

**Comment by catalin-ilea** [comment]
lgtm

File: varnish-sidekick@.service [hunk 1]
```diff
@@ -5,15 +5,17 @@ After=varnish@%i.service
 
 [Service]
 ExecStart=/bin/sh -c "\
-  export VARNISH_IP=$HOSTNAME; \
+  etcdctl set   /ft/services/varnish/healthcheck true; \
+  etcdctl mkdir /ft/services/varnish/servers; \
+  etcdctl set /ft/healthcheck/varnish-%i /__health; \
+  etcdctl set /vulcand/frontends/varnish-public-endpoint/frontend '{\"Type\":\"http\", \"BackendId\":\"vcb-varnish\", \"Route\":\"PathRegexp(`/.*`) && MethodRegexp(`GET|HEAD`) && HostRegexp(`read-.*-up.ft.com`)\"}'; \
```
Comment (Zapadlo) [inline varnish-sidekick@.service]: Need to escape dots in HostRegexp `read-.*-up\.ft\.com`

File: varnish-sidekick@.service [hunk 2]
```diff
@@ -5,15 +5,17 @@ After=varnish@%i.service
...
   while true; do \
-    export VARNISH_PORT=$(/usr/bin/docker port varnish-%i 80 | cut -d':' -f2); \
-    etcdctl set /services/varnish/%i/host $VARNISH_IP --ttl 60; \
-    etcdctl set /services/varnish/%i/port $VARNISH_PORT --ttl 60; \
-    sleep 45; \
+    export PORT=$(/usr/bin/docker port varnish-%i 80 | cut -d':' -f2); \
```
Comment (Zapadlo) [inline varnish-sidekick@.service]: I think we should move this command outside the while loop. There is no way service can change the port on the machine within the lifecycle of the sidekick.

Comment (tosan88) [inline varnish-sidekick@.service]: I can move this one, but other services do the same thing.

File: varnish-sidekick@.service (hunk 1 above)
Comment (tosan88) [inline varnish-sidekick@.service]: Right, I'm doing it :)

File: varnish-sidekick@.service (hunk 2 above)
Comment (Zapadlo) [inline varnish-sidekick@.service]: Yep, we'll move then in due time, but this service is also one of few that sleep for only 4 seconds.

**Comment by Zapadlo** [comment]
+1

**Comment by martingartonft** [comment]
+1

**Pull request merged**